Docs/developer_guide/parameter_nodes.md

```diff
@@ -293,7 +299,7 @@ This is the chosen behavior for the following reasons:
 
 Because Python objects are returned by reference, when a cached value is returned and then modified, the modification needs to be written back to the parameter node. Otherwise, the cached value and the parameter node will get out of sync.
 
-This write-on-change behavior has been implemented for the ListSerializer. The ListSerializer does not actually return a `list`, it returns an `ObservedList` that updates the parameter node whenever it is modified. `ObservedList` implements most `list` functions. This allows the following to work seamlessly:
+This write-on-change behavior has been implemented for the serializers for `list`, `tuple`, `dict`, and the `parameterPack`s. The ListSerializer does not actually return a `list`, it returns an `ObservedList` that updates the parameter node whenever it is modified. `ObservedList` implements most `list` functions. This allows the following to work seamlessly:
```

Comment:
`parameterPack` is missing from this PR.

Reply:
Good catch. Will fix docs. For reference for anyone else, `parameterPack` will be a part of a subsequent PR around the parameter node wrapper.

Reply:
Fixed.